examples/multi-layer/chile2010_ml/Makefile

```diff
@@ -0,0 +1,111 @@
+# Makefile for Clawpack code in this directory.
```

Comment:
Probably do not need `_ml` directory name given it's in the multi-layer directory.